core/src/org/sbml/jsbml/SBase.java

```diff
@@ -697,6 +697,11 @@
    */
   @Override
   public int hashCode();
+  
+  /**
+   * Returns {@code true} if the given extension exists in the model
```

Comment:
See my other comment, the extension can exist directly regarding the current object.